test/TEST_REVIEW_INDEX.md

```
@@ -192,16 +192,14 @@ usually be higher-leverage than file-by-file fixes.
 | U39 | journal/repository | test/features/journal/repository/TEST_REVIEW.md | ✅ |
 | — | **journal feature index** | test/features/journal/TEST_REVIEW.md | ✅ |
 
-### daily_os / daily_os_next
+### daily_os_next
 | Unit | Scope | Report | Status |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)